### DISCUSSION TITLE
JPG converted from RAW are all black

### DISCUSSION THREAD

**Discussion by panzeyu2013** [photoprism/photoprism#4659] [Q&A] [answered]
I use the indexing function of the photoprism and it automatically convert all raw files into jpg format. However, all these jpgs are black.

My suffix of raw file is .nef high efficient star from Nikon Z8. It seems that this codec is supported by photoprism from some official document.

**Comment by panzeyu2013** [comment]
By the way I have a question. If I click complete rescan, will the existing converted jpgs be replaced with new converted version?

**Comment by omerdduran** [comment]
Hi. If you're seeing black JPEGs from your NEF files, I recommend going through our troubleshooting checklist here: https://docs.photoprism.app/getting-started/troubleshooting/#broken-thumbnails A full rescan of your library will re-index all files, but will not recreate thumbnails that already exist. If you need to regenerate all thumbnails, you should instead use the `photoprism thumbs -f` command, which will force PhotoPrism to create new versions of all thumbnails.

**Reply by panzeyu2013** [reply]
I tested the conversion of ARW files from Sony Cameras, it works just fine. I assume the support for high efficient raw from Nikon is not complete. Thanks for your reply!

**Reply by omerdduran** [reply, marked as answer]
PhotoPrism uses Darktable or RawTherapee for RAW conversion. If your format isn't working, it may not be supported by either tool yet. You can try our preview build with the newer Darktable version to see if this resolves the issue.

https://docs.photoprism.app/user-guide/settings/advanced/#file-conversion

**Reply by panzeyu2013** [reply]
I went through some document from RawTherapee and Darktable. The high efficient and high efficient star codec is not supported by both these two, even in the latest version. Thanks again for helping.
